sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -25,6 +25,658 @@ python -m pip install azure-ai-documentintelligence
 - You need an [Azure subscription][azure_sub] to use this package.
 - An existing Azure AI Document Intelligence instance.
 
+#### Create a Cognitive Services or Document Intelligence resource
+
+Document Intelligence supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Document Intelligence access only, create a Document Intelligence resource. Please note that you will need a single-service resource if you intend to use [Azure Active Directory authentication](#create-the-client-with-an-azure-active-directory-credential).
+
+You can create either resource using: 
+
+* Option 1: [Azure Portal][cognitive_resource_portal].
+* Option 2: [Azure CLI][cognitive_resource_cli].
+
+Below is an example of how you can create a Document Intelligence resource using the CLI:
+
+```PowerShell
+# Create a new resource group to hold the Document Intelligence resource
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
+
+```PowerShell
+# Create the Document Intelligence resource
+az cognitiveservices account create \
```

Comment:
This comment should be addressed for a future release:

We should verify if these are still the recommended commands for creating a resource.